tests/protocol_test.go

```diff
@@ -1,50 +0,0 @@
-package client
-
-import (
-	"fmt"
-	"testing"
-
-	"github.com/google/uuid"
-	"github.com/stretchr/testify/assert"
-	xlineapi "github.com/xline-kv/go-xline/api/xline"
-	"github.com/xline-kv/go-xline/client"
-	"github.com/xline-kv/go-xline/xlog"
-	"go.uber.org/zap/zapcore"
-)
-
-func TestCurp(t *testing.T) {
```

Comment:
We could also add some tests for protocol client here, instead of removing it completely.

Reply:
In this modification, the method propose in protocol clinet has been made private, so it cannot be tested separately.